src/components/Router.js

```diff
@@ -52,6 +53,23 @@ export default class Router extends Component {
     initNavPaths()
   }
 
+  componentDidUpdate(prevProps, prevState) {
+    const { pathname, search } = this.state
+    if (prevState.pathname === pathname) return
+
+    const handler = Nav.findHandler(pathname)
+
+    if (handler && handler.title) {
+      if (_.isFunction(handler.title)) {
+        document.title = handler.title(Nav.getHandlerProps(handler, pathname, search))
+      } else {
+        document.title = handler.title
```

Comment:
Should this be:
```javascript
document.title = handler.title + ' | Saturn'
```
?

Reply:
At least in theory, that's what favicons are for